src/books/books.controller.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { BooksService } from './books.service';
+import { BooksCreateDto } from './dto/books.create.dto';
+import { BooksUpdateDto } from './dto/books.update.dto';
+
+@Controller('books')
+export class BooksController {
+  constructor(private readonly booksService: BooksService) {}
+
+  @Get()
+  async findAll() {
```

Comment:
Luôn ghi rõ sẽ trả về cái gì. Ở đây mình trả về book item, không trả về trực tiếp book/book entity

```suggestion
  async findAll(): Promise<BookItem[]> {
```